mod-orders-storage/examples/acquisitions_unit_collection.sample

```diff
@@ -0,0 +1,20 @@
+{
+   "acquisitionsUnits":
+     [
+       {
+          "name": "main",
+          "protectCreate": "true",
```

Comment:
Please change all the values to boolean instead of string. Please refer to [Understanding json schema - boolean](https://json-schema.org/understanding-json-schema/reference/boolean.html)

Reply:
done

Reply:
Thank you for comment this is really helpful.